### DISCUSSION TITLE
Agg function error

### DISCUSSION THREAD

**Discussion by rrdvg** [vaexio/vaex#1139] [General]
I am trying to do sum aggregate CY = current year sales by date

![image](https://user-images.githubusercontent.com/76768823/103336927-84afd900-4a2e-11eb-9471-a7d03c257205.png)

It says data type is string I tried converting it to int as well, did not work

df_group = df.groupby(df.Reporting_Date, agg=vaex.agg.sum(df.CY))
df_group

Getting below error 
AttributeError: type object 'str' has no attribute 'kind'

Any help appreciated

**Comment by rrdvg** [comment]
issue is resolved. There was an issue with data

**Reply by maartenbreddels** [reply]
Great, thanks for reporting back!
